src/codegen/src/main/scala/PySparkWrapperTest.scala

```diff
@@ -270,7 +270,7 @@ class PySparkTransformerWrapperTest(entryPoint: Transformer,
         case "ValueIndexerModel" => valueIndexerModelString(entryPointName)
         case "_CNTKModel" | "FastVectorAssembler" | "MultiNGram" | "ImageFeaturizer" | "_ImageFeaturizer"
            | "_ImageTransformer" | "UnrollImage" | "HashTransform" | "Timer"
-           | "StopWordsRemoverTransform"  | "ImageSetAugmenter"
+           | "StopWordsRemoverTransform"  | "ImageSetAugmenter" | "PowerBITransformer"
```

Comment:
Any way we can have a "contrib" section where something like this goes? I would like to avoid adding dependencies like PowerBI into the core library.